Earthfile

```diff
@@ -1,3 +1,5 @@
+VERSION 0.6
```

Comment:
Yay! We should probably do this for all our Earthfiles. But that could be a separate PR

Reply:
and converting `--build-args` too :tada: